### PR TITLE
Add connectRouter + example

### LINKED ISSUE #1: Time Travelling does not work
With Redux DevTools, everything seems to work well except time travelling.

### PR DESCRIPTION
**Pull request by dictions**
Fixes #1

### CODE REVIEW

File: examples/src/reducers/root.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
 import { combineReducers } from 'redux'
-import { routerReducer } from 'connected-react-router'
 import counterReducer from './counter'
 
 const rootReducer = combineReducers({
-  count: counterReducer,
-  router: routerReducer,
+  count: counterReducer
```
Comment (supasate) [inline examples/src/reducers/root.js]: Add trailing comma to pass eslint.

**Comment by supasate** [comment]
Need to modify ConnectedRouter.js also

**Comment by supasate** [comment]
@dictions Need another little linting fix about trailing comma  (https://github.com/supasate/connected-react-router/pull/7#pullrequestreview-14317484)

**Comment by dictions** [comment]
@supasate weird, I couldn't get eslint to fail with or without the trailing comma.

**Comment by supasate** [comment]
Oh, I just noticed that I don't use airbnb rules in this project. My eyes worked like airbnb linting compiler lol.

**Comment by supasate** [comment]
LGTM. Thank you for your contribution!

**Comment by dictions** [comment]
Of course! Mind bumping the version on merge?

**Comment by supasate** [comment]
Yes. I'll add some more details in the README before bumping version.

**Comment by supasate** [comment]
Already released in v2.0.0-alpha.1 https://github.com/supasate/connected-react-router/releases

**Comment by wtgtybhertgeghgtwtg** [comment]
Why is it a higher-order reducer?

**Comment by supasate** [comment]
@wtgtybhertgeghgtwtg To initialize router state with current browser history. Additional benefit is users don't have to mount router reducer themselves coz it's automatically mounted.

**Comment by wtgtybhertgeghgtwtg** [comment]
It complicates `replaceReducer`.
```
const newReducer = combineReducers(reducers);
store.replaceReducer(newReducer);
```
would have to be replaced with
```
const newReducerWithoutRouting = combineReducers(reducers);
const newReducer = connectReducer(newReducerWithoutRouting);
store.replaceReducer(newReducer);
```

**Comment by supasate** [comment]
I see your point about replacing reducer. Without higher order reducer, we've ever tried with following methods. 

1. Dispatching `onLocationChanged` in `constructor` of `ConnectedRouter` with the current history location. 
However, the initial rendered page will be the default state before transitioning to the page of current history location. Therefore, we need somehow to set initial reducer state with values in `history` object. 

2. We initially had an idea about creating a reducer creator that takes `history` object. Then, users call that creator and mount it to the root reducer. 
However, we found it complicated when most users separate the root reducer to another file and need to make a root-reducer creator to take `history` object in.

3. Our next solution is to import `history` directly in our router reducer file to create initial state. However, that makes it hard to test because it's highly coupled with `history` object.

4. We've reached the current solution to solve above issues by using higher order reducer.

(see our discussion [here](https://github.com/supasate/connected-react-router/issues/1))

Therefore, I think, currently, I choose to trade-off with a bit complication of replacing reducer until we find a better solution. If you have any idea, please let me know.

**Pull request merged**